src/UI/Cells/FileLinkCell.js

```diff
@@ -0,0 +1,19 @@
+var vent = require('vent');
+var Backgrid = require('backgrid');
+
+module.exports = Backgrid.Cell.extend({
+    className : 'file-link-cell',
+
+    render : function() {
+        // Network accessible path to /mnt directory
+        var replace = /^\/mnt\//;
```

Comment:
As I wrote in an outdated diff: `These variables are of course dependent on the user's setup, and must be configurable.`. So the user would have to discover the correct replacement themselves.